spec/controllers/publishers/vacancies/base_controller_spec.rb

```diff
@@ -9,7 +9,7 @@
 
       it "does perform the task" do
         expect(UpdateGoogleIndexQueueJob).to receive(:perform_later)
-        controller.update_google_index(vacancy)
+        controller.send(:update_google_index, vacancy)
```

Comment:
Not sure why!

Reply:
I made the methods in the base controller private, because there should be no public methods in a controller that you don't want to be actions.

Having made them private, we need to `send` in the tests in order to access them.  They were AFAICT only public to allow this testing.

A better way would be to properly subclass the base controller in the tests, but this felt good enough for now.

Reply:
Makes sense!